devimint/src/tests.rs

```diff
@@ -2275,6 +2318,40 @@ pub async fn handle_command(cmd: TestCmd, common_args: CommonArgs) -> Result<()>
             let dev_fed = dev_fed(&process_mgr).await?;
             cannot_replay_tx_test(dev_fed).await?;
         }
+        TestCmd::UpgradeTest { old_fedimintd } => {
+            let (process_mgr, _) = setup(common_args).await?;
+            std::env::set_var("FM_FEDIMINTD_BASE_EXECUTABLE", old_fedimintd);
+            let mut dev_fed = dev_fed(&process_mgr).await?;
+            let client = dev_fed.fed.new_joined_client("test-client").await?;
+
+            tokio::try_join!(stress_test_fed(&dev_fed), wait_session(&client))?;
+
+            info!("shutting down servers");
+            dev_fed.fed.terminate_server(3).await?;
+            // kill 3rd peer early
+            tokio::time::sleep(Duration::from_secs(30)).await;
```

Comment:
Linter's failing, we can use `fedimint_core::task::sleep_in_test` instead of `tokio::time::sleep`.